infra/modules/aws-psoxy-lambda/main.tf

```diff
@@ -9,15 +9,16 @@ terraform {
 }
 
 resource "aws_lambda_function" "psoxy-instance" {
-  function_name    = var.function_name
-  role             = aws_iam_role.iam_for_lambda.arn
-  architectures    = ["arm64"] # 20% cheaper per ms exec time than x86_64
-  runtime          = "java11"
-  filename         = var.path_to_function_zip
-  source_code_hash = var.function_zip_hash
-  handler          = var.handler_class
-  timeout          = var.timeout_seconds
-  memory_size      = var.memory_size_mb
+  function_name                  = var.function_name
+  role                           = aws_iam_role.iam_for_lambda.arn
+  architectures                  = ["arm64"] # 20% cheaper per ms exec time than x86_64
+  runtime                        = "java11"
+  filename                       = var.path_to_function_zip
+  source_code_hash               = var.function_zip_hash
+  handler                        = var.handler_class
+  timeout                        = var.timeout_seconds
+  memory_size                    = var.memory_size_mb
+  reserved_concurrent_executions = var.reserved_concurrent_executions
```

Comment:
new, to be able to limit

Reply:
weird name for this, but I see comes from AWS

my suggestion is to put the `coalesce(var.reserved_concurrent_executions, -1)` - so that either leaving `undefined` or passing as `null` will have same effect; that seems most defensive imho